api/comments/serializers.py

```diff
@@ -47,12 +51,31 @@ class CommentSerializer(JSONAPISerializer):
     class Meta:
         type_ = 'comments'
 
+    def get_is_abuse(self, obj):
+        auth = Auth(self.context['request'].user)
```

Comment:
Is it necessary to construct an `Auth` object here? This method only seems to need a `user`.